azure_pipelines/spm-framework.yml

```diff
@@ -367,7 +373,7 @@ jobs:
       targetType: 'inline'
       script: |
         # NOTE : This should be the last step since it changes branch from main to $(docsRepositoryBranch)
-        if [ ! -d "docs.temp" ]; then
```

Comment:
Jazzy creates the documents under `docs` now rather than `docs.temp/docs` hence the change here and below

Reply:
Hey Silviu, can you clarify a little bit more this point? From the build_docs.sh script I can't see that the we specify the name of the destination folder to "docs" instead of "docs.temp". I'd personally prefer to still use the newly create "docs.temp". Do you know if we can specify the "docs.temp" as destination folder of the jazzy command?

Reply:
Actually that makes more sense :) Changed jazzy output folder